ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/ListReceiptsTest.java

```diff
@@ -0,0 +1,150 @@
+                .getResourceContent("authentication_token_response.json")).mock();
+    }
+
+    @After
+    public void cleanup() {
+        RepositoryFactory.clearInstance();
+    }
+
+    @Test
+    public void testListReceipts_userHasMultiplereceipts() {
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("receipt_list_response.json")).mock();
+
+        // run test
+        mActivityTestRule.launchActivity(null);
+
+        // assert
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar))))
+                .check(matches(withText(R.string.title_activity_receipt_list)));
+        onView(withId(R.id.list_receipts)).check(matches(isDisplayed()));
+
+        onView(withId(R.id.list_receipts))
+                .check(matches(atPosition(0, hasDescendant(withText("November 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.credit)))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("Payment")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("+ 20.00")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("November 01, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(0, hasDescendant(withText("USD")))));
+
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.credit)))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("Payment")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("+ 25.00")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("November 02, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(1, hasDescendant(withText("CAD")))));
+
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2,
+                hasDescendant(withText(com.hyperwallet.android.receipt.R.string.debit)))));
+        onView(withId(R.id.list_receipts)).check(
+                matches(atPosition(2, hasDescendant(withText("Card Activation Fee")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2, hasDescendant(withText("- 1.95")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2, hasDescendant(withText("November 10, 2017")))));
+        onView(withId(R.id.list_receipts)).check(matches(atPosition(2, hasDescendant(withText("USD")))));
+
+        onView(withId(R.id.list_receipts))
+                .check(matches(atPosition(3, hasDescendant(withText("December 2017")))));
```

Comment:
Transactions should occur in order createdOn descending.  So December transactions should appear before November. Also can you update your test data to use 2019 instead of 2017?

Reply:
@skoong Done